samples/src/JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -98,6 +101,27 @@ private static async Task Run()
                             x.ForTopic<OrderOnItsWayEvent>(cfg =>
                                 cfg.WithReadConfiguration(rc =>
                                     rc.WithSubscriptionGroup("GroupB")));
+
+
+                            // New proposal
+                            x.ForQueue<OrderPlacedEvent>(QueueAddress.FromArn("arn:aws:sqs:eu-west-1:111122223333:queue1"));
```

Comment:
Just spitballing without digging further, what about as well or instead for the signature:

```csharp
x.ForQueueArn<OrderPlacedEvent>("arn:aws:sqs:eu-west-1:111122223333:queue1")
```

Just thinking from an API usability/readability point-of-view.

Reply:
Yeah, I'm with you on this, I think it's a little clearer rather than with overloads.
Would like to hear other views.